client/src/Donations.js

```diff
@@ -15,7 +15,9 @@ const Donations = () => {
     <p>Sort Code: 12-34-56</p>
     <p>Alternatively, you can also donate to us via Just Giving - click the button below to be taken to their website.</p>
     <img className="dImage" src={image} alt="group of children" />
-      <a href="https://www.justgiving.com/"><button className="button">Thank you for your Donation</button></a>
+      <a href="//widgets.justgiving.com/Button/Redirect?p=eyJUZXh0IjoiQ29kZSBZb3VyIEZ1dHVyZSIsIklkIjoiN2I4MDA2YjQtZjk2OC00NDdjLTk1NDEtMDA3MTg3MTkxZTU3IiwiQ2hhcml0eUlkIjoyNzg4NDg3LCJTaXplIjoibCIsIlJlZmVyZW5jZSI6IjEyMyIsIlR5cGUiOiJDaGFyaXR5RG9uYXRlIn0=">
+        <img src="//widgets.justgiving.com/Button?p=eyJUZXh0IjoiQ29kZSBZb3VyIEZ1dHVyZSIsIklkIjoiN2I4MDA2YjQtZjk2OC00NDdjLTk1NDEtMDA3MTg3MTkxZTU3IiwiQ2hhcml0eUlkIjoyNzg4NDg3LCJTaXplIjoibCIsIlJlZmVyZW5jZSI6IjEyMyIsIlR5cGUiOiJDaGFyaXR5RG9uYXRlIn0=" alt="Donation Button" />
+        </a>
     </div>
```

Comment:
can you please add a video too? i tried but did not work. if i use iframe tag, it only works when using url. when i use video tag with mp4 file, it display a blank video

Reply:
We're happy to rely on the video component so we don't need a change for now